### PR TITLE
[ES] Add `People/Nomination Assessment Team/NAT meetings`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Nomination_Assessment_Team/NAT_meetings/es.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+### Temas descartados
+
+Estos temas eran demasiado complejos para tratarlos en una reunión por diversas razones, pero pueden llegar a ser temas viables en el futuro:
+
+| Tema | Motivo del descarte |
+| :-- | :-- |
+| El trabajo de los BN y el NAT debe ser menos robótico | Debe dividirse en problemas específicos y sugerencias |
+| Recompensas significativas por el trabajo de los BN y el NAT | Fuera del control del NAT |
+| Dejar que los BN influyan en las solicitudes y/o en las evaluaciones actuales de los BN | El alcance es muy amplio y debería hacerse primero una propuesta por escrito |
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/People/Nomination_Assessment_Team/NAT_meetings/es.md:44]:
```suggestion
| Dejar que los BN influyan en las solicitudes y/o en las evaluaciones de los BN actuales | El alcance es muy amplio y debería hacerse primero una propuesta por escrito |
```

**Review by Unnombredeusuarioextremadamentelargo — CHANGES_REQUESTED** [review]

File: wiki/People/Nomination_Assessment_Team/NAT_meetings/es.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+tags:
+  - NAT/BN meetings
+  - Nomination Assessment Team meetings
+  - Reuniones del NAT
+  - Reuniones de los BN
+  - Reuniones del Nomination Assessment Team
+  - Reuniones del equipo de evaluación de nominaciones
```
Comment (agatemosu) [inline wiki/People/Nomination_Assessment_Team/NAT_meetings/es.md:8]: Reuniones en minúsculas

File: wiki/People/Nomination_Assessment_Team/NAT_meetings/es.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+# Reuniones del NAT
+
+Las **reuniones del NAT** son discusiones directas organizadas por el [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (*NAT*) en el canal `#modding` del [servidor de Discord osu!](/wiki/Community/osu!_Discord_server). Los temas de las reuniones están relacionados con el ecosistema del [mapping](/wiki/Beatmapping) y el [modding](/wiki/Modding), e incluyen problemas, posibles soluciones, nuevas ideas e informes de progreso. Si bien la mayoría de los que contribuyen a las discusiones son los miembros del NAT o los [Beatmap Nominators](/wiki/People/Beatmap_Nominators), cualquiera puede participar.
```
Comment (agatemosu) [inline wiki/People/Nomination_Assessment_Team/NAT_meetings/es.md:13]:
```suggestion
Las **reuniones del NAT** son discusiones directas organizadas por el [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (*NAT*) en el canal `#modding` del [servidor de Discord osu!](/wiki/Community/osu!_Discord_server). Los temas de las reuniones están relacionados con el ecosistema de [mapping](/wiki/Beatmapping) y [modding](/wiki/Modding), e incluyen problemas, posibles soluciones, nuevas ideas e informes de progreso. Si bien la mayoría de los que contribuyen a las discusiones son los miembros del NAT o los [Beatmap Nominators](/wiki/People/Beatmap_Nominators), cualquiera puede participar.
```

File: wiki/People/Nomination_Assessment_Team/NAT_meetings/es.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+Las reuniones suelen anunciarse con más de una semana de anticipación en el canal `#modding`.
```
Comment (agatemosu) [inline wiki/People/Nomination_Assessment_Team/NAT_meetings/es.md:15]:
```suggestion
Las reuniones suelen anunciarse con más de una semana de antelación en el canal `#modding`.
```

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**